Connection.php

```diff
@@ -391,10 +407,20 @@ public function executeCommand($name, $params = [])
             $command .= '$' . mb_strlen($arg, '8bit') . "\r\n" . $arg . "\r\n";
         }
 
-        \Yii::trace("Executing Redis Command: {$name}", __METHOD__);
-        fwrite($this->_socket, $command);
-
-        return $this->parseResponse(implode(' ', $params));
+        // Retries
+        $rcount = 0;
+        $lex = NULL;
+        while ($rcount < $this->retriesmax) {
+            try {
+                \Yii::trace("Executing Redis Command: {$name} | Try {$rcount}", __METHOD__);
+                fwrite($this->_socket, $command);
+                return $this->parseResponse(implode(' ', $params));
+            } catch (Exception $ex) {
```

Comment:
the exception caught should distinguish between a connection failure and an error returned by redis in which case a retry would fail the same way on other instances too as it is done in #106